bootstrap/pom.xml

```diff
@@ -20,8 +20,8 @@
 
     <properties>
         <maven.deploy.skip>true</maven.deploy.skip>
-        <maven.compiler.source>1.6</maven.compiler.source>
-        <maven.compiler.target>1.6</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
It is time to set "new" standards. Didn't build without it. Maven complained and recommended at least 1.7 but I think 1.8 should be best for MC.

Reply:
I'd split this up into separate PR.

Reply:
That is java 6 only to warn users that they need java 8